index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Medical core</title>
+  </head>
+  <body class="body">
+    <div class="navbar">
+      <header class="header">
+        <h1 class="header__logo">Medical Core</h1>
+        <ul class="header__navbar">
+          <li class="header__item">Home</li>
+          <li class="header__item">Product</li>
+          <li class="header__item">Pricing</li>
+          <li class="header__item">Contact</li>
+        </ul>
+
+        <ul class="header__leftbtn">
+          <li class="btn">Login</li>
+          <li class="btn btn--active">Join Us</li>
+        </ul>
+      </header>
+
+      <div class="banner">
+        <h4 class="banner__header">Welcome</h4>
+        <p class="banner__description">A Great Place to Receive Care</p>
+        <p class="banner__moto">Get your best looking smile now!</p>
+
+        <ul class="banner__btn">
+          <li class="btnn btn--activee">Join Us</li>
+          <li class="btnn btn--disactive">Learn More</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="expert">
+      <img src="./assets/main-image.png" alt="expert" class="expert__img" />
+
+      <div class="expert__body">
+        <div class="expert__border"></div>
+        <h2 class="expert__title">Meet Our Experts</h2>
+        <p class="expert__description">
+          Problems trying to resolve the conflict between the two major realms
+          of Classical physics: Newtonian mechanics
+        </p>
+        <p class="expert__link">Learn More</p>
+      </div>
+    </div>
+
+    <div class="team">
+      <h5 class="banner__header">Team</h5>
```

Comment:
`banner__header` is an element of `banner` not `team`. use `team__header` in `team` section